.github/workflows/main.yml

```diff
@@ -70,9 +70,9 @@ jobs:
       run: |
         nuget setapikey ${{ secrets.MYGET_APIKEY }} -source ${{ secrets.MYGET_URL }}
         foreach ($file in Get-ChildItem -Path "${{ env.telemetrysdk_path }}\*" -File -Include *.nupkg) {
-          nuget push $file.fullname -Source ${{ secrets.MYGET_URL }} }
+          nuget push $file.fullname -Source ${{ secrets.MYGET_URL }} } -NoSymbols
```

Comment:
Few things
* `NoSymbols` is declared outside he foreach block so this won't run
* If `nuget push` by default pushes symbols, then how about we just do both in one step
* Please test these commands before we merge this PR.

Reply:
I did some tests and nuget was not automatically pulling in symbols locally, at least for me.  I think going with the split is still the safest approach.